inst/livy/spark-1.5.2/utils.scala

```diff
@@ -423,10 +423,10 @@ object Utils {
   }
 
   def zipDataFrames(sc: SparkContext, df1: DataFrame, df2: DataFrame) : DataFrame = {
-      val sqlContext = new org.apache.spark.sql.SQLContext(sc)
-      sqlContext.createDataFrame(
-        df1.rdd.zip(df2.rdd).map { case (r1, r2) => Row.merge(r1, r2) },
-        StructType(df1.schema ++ df2.schema))
+    val spark = SparkSession.builder().getOrCreate()
```

Comment:
@javierluraschi What is the correct way to update functions in `utils.scala`? Tried to create the file with same name under `spark-2.0.0/`, but it does not work.